RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -162,7 +157,7 @@ private static PluginManifest CreatePluginDefinition(IPackage package)
             pluginDefn.Developers = GetValidManifestString(ListToString(package.Authors));
 
             pluginDefn.Homepage = GetValidManifestString(package.ProjectUrl?.ToString());
-            pluginDefn.Key = GetValidManifestString(package.Id) + PluginKeySuffix;
```

Comment:
Plugin key no longer needs to be suffixed (although perhaps it would be useful to have a suffix that indicates it was generated by the SDK - or even a have a property definition that indicates which version of the SDK generated it).